strings/french.strings

```diff
@@ -58,64 +58,64 @@
+"Only the group creator can remove members." = "Seul le fondateur du groupe peut exclure des membres.";
 
 /* MISSING TRANSLATION - model/contracts/group.js */
-"There is an identical open proposal." = "There is an identical open proposal.";
+"There is an identical open proposal." = "Une proposition ouverte identique existe déjà.";
 
 /* MISSING TRANSLATION - model/state.js */
-"Automated ban because they're sending malformed messages resulting in: {error}" = "Automated ban because they're sending malformed messages resulting in: {error}";
+"Automated ban because they're sending malformed messages resulting in: {error}" = "Bannissement automatique pour envoi de messages malformés causant l'erreur : {error}";
 
 /* MISSING TRANSLATION - views/components/ProfileCard.vue */
-"Add a bio" = "Add a bio";
+"Add a bio" = "Ajouter une bio";
 
 /* MISSING TRANSLATION - views/components/ProfileCard.vue */
-"Add payment information" = "Add payment information";
+"Add payment information" = "Indiquer vos informations de paiement";
 
 /* MISSING TRANSLATION - views/components/ProfileCard.vue */
-"Close profile" = "Close profile";
+"Close profile" = "Fermer le profil";
 
 /* MISSING TRANSLATION - views/components/ProfileCard.vue */
-"Edit bio" = "Edit bio";
+"Edit bio" = "Éditer la bio";
 
 /* MISSING TRANSLATION - views/components/ProfileCard.vue */
-"Edit payment info" = "Edit payment info";
+"Edit payment info" = "Éditer les informations de paiement";
 
 /* MISSING TRANSLATION - views/components/ProfileCard.vue */
-"Help other users send monthly contributions your way by adding your payment information." = "Help other users send monthly contributions your way by adding your payment information.";
+"Help other users send monthly contributions your way by adding your payment information." = "Aidez les autres utilisateurs à vous envoyer des contributions mensuelles en indiquant vos informations de paiement.";
 
 /* MISSING TRANSLATION - views/components/ProfileCard.vue */
 "No longer a member of the group" = "No longer a member of the group";
 
 /* MISSING TRANSLATION - views/components/ProfileCard.vue */
-"Send message" = "Send message";
+"Send message" = "Envoyer un message";
 
 /* MISSING TRANSLATION - views/components/ProfileCard.vue */
-"Show profile" = "Show profile";
+"Show profile" = "Afficher le profil";
 
 /* MISSING TRANSLATION - views/components/ProfileCard.vue */
-"{username} profile" = "{username} profile";
+"{username} profile" = "Profil de {username}";
 
 /* MISSING TRANSLATION - views/components/ProfileCard.vue, views/pages/DesignSystem.vue */
-"Remove member" = "Remove member";
+"Remove member" = "Exclure un membre";
 
 /* MISSING TRANSLATION - views/components/VotingRulesInput.vue */
-"\"No\" votes required to block a proposal" = "\"No\" votes required to block a proposal";
+"\"No\" votes required to block a proposal" = "Nombre de votes négatifs requis pour bloquer une proposition";
```

Comment:
Aucun vote négatif n'est requis pour bloquer une proposition

Reply:
I'm a bit unsure about this one. The text is displayed for a slider that can only accept values between 1 and 60, so at least one negative vote is always required.

Besides, if your interpretation is correct; then the quotes around the `"No"` in the English version are very confusing, and should be removed I think.

Reply:
You're right, I didn't see that one in its context.
I think that the english version is a bit awkward then. @taoeffect ?

Reply:
I think the idea was, `1 "no" votes required to block a proposal`, `2 "no" votes required to block a proposal`, etc.

So if whatever you write makes the same sense in French it's fine. Yes I agree it's a bit awkward but it works.

Reply:
All right then! We can close that one